azkaban-execserver/src/main/java/azkaban/execapp/ServerStatisticsServlet.java

```diff
@@ -73,7 +73,7 @@ protected void fillRemainingMemoryPercent(ExecutorInfo stats){
         && new File("/bin/grep").exists()
```

Comment:
This is a pretty long function, and this if condition spans the entire method. It would be better to check the negation of this condition and return early:

```java
if (!new File("/bin/bash").exists() ||
    !new File("/bin/cat").exists() ||
    !new File("/bin/grep").exists() ||
    !new File("/proc/meminfo").exists()) {
  return;
}
...
```